glib/src/boxed_inline.rs

```diff
@@ -29,7 +29,7 @@ macro_rules! glib_boxed_inline_wrapper {
         impl $(<$($generic $(: $bound $(+ $bound2)*)?),+>)? std::marker::Copy for $name $(<$($generic),+>)? {}
 
         $crate::glib_boxed_inline_wrapper!(
-            @generic_impl [$($attr)*] $name $(<$($generic $(: $bound $(+ $bound2)*)?),+>)?, $ffi_name,
+            @generic_impl $name $(<$($generic $(: $bound $(+ $bound2)*)?),+>)?, $ffi_name,
```

Comment:
Removing this means you won't be able to add documentation anymore for these types. Is it expected?

Reply:
Yes, because it's completely unused in that macro call. The documentation is already added in another place.

We unnecessarily passed the attributes into this macro call, i.e. they are not actually used in there because they're already used up elsewhere.

Reply:
No `repr(c)` or `repr(transparent)` either?

Reply:
Same answer: that's already handled before (and in this case it is always `#[repr(transparent)]` anyway)

Reply:
Look in line 12/13

Reply:
Perfect then!